package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "norwegian-utils",
-  "version": "0.3.1",
+  "version": "0.4.0",
```

Comment:
hmmm no?..

Reply:
Yea, 0.3.2 is better.

Reply:
0.4.0. is the last version on npm, so I guess 0.4.1 is correct?